nuget/spec/dependabot/nuget/file_fetcher_spec.rb

```diff
@@ -9,6 +9,18 @@
 require_common_spec "file_fetchers/shared_examples_for_file_fetchers"
 
 RSpec.describe Dependabot::Nuget::FileFetcher do
+  before do
+    allow(file_fetcher_instance).to receive(:commit).and_return("sha")
+
+    stub_request(:get, File.join(url, ".config?ref=sha"))
+      .with(headers: { "Authorization" => "token token" })
+      .to_return(
+        status: 404
+      )
+  end
+
+  before { allow(file_fetcher_instance).to receive(:commit).and_return("sha") }
+
```

Comment:
```suggestion
```
This setup already happens in the `before` block above.

Reply:
it's the blank line that was needed.
 rubocop fixes.

Reply:
I'm not sure I follow. You have the same setup code happening twice here, so I was suggesting removing the duplicated part.

Reply:
updated code.  I missed this one as it was part of a merge-commit conflict resolution.